controllers/__init__.py

```diff
@@ -92,14 +92,36 @@ def upsert_evc(self, evc: Dict) -> Optional[Dict]:
         updated = self.db.evcs.find_one_and_update(
             {"_id": evc["id"]},
             {
-                "$set": model.dict(exclude={"inserted_at"}, exclude_none=True),
+                "$set": model.dict(
+                    exclude={"inserted_at"},
+                    exclude_none=True),
                 "$setOnInsert": {"inserted_at": utc_now},
             },
             return_document=ReturnDocument.AFTER,
             upsert=True,
         )
         return updated
 
+    def update_evc(self, evc: Dict) -> Optional[Dict]:
+        """Update an EVC.
+        This is needed to correctly set None values to fields"""
+
+        # Check for errors in fields only.
+        EVCBaseDoc(
+            **{
+                **evc,
+                **{"_id": evc["id"]}
+            }
+        )
```

Comment:
@Alopalao, unfortunately this backfired when trying to set `queue_id: 1` from an EVC without any queue_id set:

![20230711_171445](https://github.com/kytos-ng/mef_eline/assets/1010796/7d4afb92-6188-41f9-8272-db59f99a340f)

Maybe you could introduce a new `EVCUpdateDoc` inhering from `EVCBaseDoc` and specifying the  [read_only_attributes](https://github.com/kytos-ng/mef_eline/blob/master/models/evc.py#L33-L40) as `Optional`? or alternatively we byte the bullet and get rid of this extra validation here. Could you double check?

Reply:
Oh. I tottally overlooked this. Yeah, will check this.

Reply:
Nice. `EVCUpdateDoc` looks good.